### PR TITLE
Allow installation on PHP 7.3

### PR DESCRIPTION
**Pull request by bsdnomad**
PHP 7.3 specified as a dependancy since officially supported by Magento 2.3.3 OS/Commerce and a default version for Magento Cloud

### CODE REVIEW

**Comment by floorz** [comment]
merge it, merge it, merge it! :)

**Comment by asrar7787** [comment]
hey guys, is there any eta when this will be merged? - thanks!

**Comment by edward-simpson** [comment]
When will this be merged?

**Comment by edward-simpson** [comment]
@bsdnomad  Why has such a simple PR not been merged yet?  This is blocking me using this module with a client, has this repo been abandoned??

**Comment by bsdnomad** [comment]
@edward-simpson I am the author of the PR, but unfortunately I neither have write access, neither ownership of the original repo so .... we wait. Fork mine and install it by composer if you want.

**Comment by DanGarner** [comment]
**The repo is effectively abandoned in my opinion.**

I escalated this through partner channels that we have, and they told me in November that it would be done mid-December - by "new developers" that they had recruited. When recently challenged why this hadn't happened, they ignored me - and we are commercial partners. I guess that they are a little embarrassed by the situation.

They do not seem to understand how real development environments work or how deployment processes rely on this kind of thing.

**Comment by edward-simpson** [comment]
Just had a chat with Loquate who said they're overhauling their entire product, and won't be making any changes to the repo til that's done

**Comment by DavidGreenwoodAYKO** [comment]
+1 for this. Anyone installing the latest Magento versions has support for PHP 7.3. PHP 7.2 stops receiving security updates and becomes unsupported in November this year. This should be a priority if you want people moving to Magento 2 to continue using your service.

**Comment by ghost** [comment]
Hi Everyone.

Reading through the comments, I can only apologise @DanGarner for the lack of communication. I used to be one of the developers who contributed to the Magento 2 Extension. I am now in a different part of the company, developing the new core engine behind the capture product. This github account has been assigned to an older email and it was only recently I was going through it and found the github mentions.

We were acquired by GB Group a couple of years ago and there has been a big overhaul of all areas within the PCA systems which has been prioritised. The integrations into 3rd parties was not priority at the time, but recently (since the new year really) they have been put on the table and how we are to tackle them.

We as developers here have put our hands up and said that we do not have the knowledge (or time) to maintain integrations for the 5-6 different platforms that we have tried to accommodate. Because of this we have started talks with a company who specialise in these sorts of integrations. Specifically the Magento 2, Salesforce Lightening and I think Dynamics 365.

To elaborate on what @edward-simpson said, Loqate is the brand that GB Group have gone with to bundle together all the location based products. There is a major overhaul of the existing Capture products from PCA as well as all the other API endpoints and services we offer. Also some of the legacy products that we have maintained for a long time now have had to be looked into.

So to summarise, we apologise for the delay and lack of communication, I will be looking at all the pull requests we have and reviewing them and I have already started to archive some of our other repo's which are not been used.

I will wrap this one off now as well!

Regards

Matt

**Comment by ghost** [comment]
> **The repo is effectively abandoned in my opinion.**
> 
> I escalated this through partner channels that we have, and they told me in November that it would be done mid-December - by "new developers" that they had recruited. When recently challenged why this hadn't happened, they ignored me - and we are commercial partners. I guess that they are a little embarrassed by the situation.
> 
> They do not seem to understand how real development environments work or how deployment processes rely on this kind of thing.

@DanGarner - I will see if I can get in touch with your partner channels here at Loqate and get them to email you with some more useful information.
